src/api/api.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+export function makeApi() {
+  return axios.create({ baseURL: "https://rickandmortyapi.com/api/" });
+}

```

Comment:
It's not neccesary to export this function because it won't be used outside this file.